tests/test_api.py

```diff
@@ -33,12 +33,18 @@ def test_basic_query(orsay_museum):
     )
 
     assert response.status_code == 200
-    assert response.json() == {
-        "id": "osm:way:63178753",
-        "label": "Musée d'Orsay (Paris)",
-        "name": "Musée d'Orsay"
-    }
 
+    resp = response.json()
+
+    assert resp['id'] == 'osm:way:63178753'
+    assert resp['name'] == "Musee d'Orsay"
+    assert resp['local_name'] == "Musée d'Orsay"
+    assert resp['class_name'] == 'museum'
+    assert resp['subclass_name'] == 'museum'
+    assert resp['address']['label'] == '62B Rue de Lille (Paris)'
+    assert resp['blocks'][0]['btype'] == 'OpeningHourBlock'
+    assert resp['blocks'][0]['is_24_7'] == False
+    assert resp['blocks'][0]['days'][2] == 'Wednesday'
```

Comment:
since we use the `lang`, it would be nice to have a test with a `lang` param

Reply:
done